plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -94,6 +94,29 @@ public static String createKuduTableForWrites(String createTable)
                 format("WITH (partition_by_hash_columns = ARRAY['%s'], partition_by_hash_buckets = 2)", column);
     }
 
+    @Test
+    public void testUnPartitionedTable()
+    {
+        String tableName = "test_unpartitioned_table" + randomNameSuffix();
+        // success create the table without partition keys
+        assertUpdate("CREATE TABLE " + tableName + " (id int WITH (primary_key=true), val varchar)");
+
+        assertQuery("SELECT COUNT(*) FROM " + tableName, "VALUES 0");
+        assertUpdate("INSERT INTO " + tableName + " VALUES (1, 'hello'), (2, 'world')", 2);
+        assertQuery("SELECT COUNT(*) FROM " + tableName, "VALUES 2");
+
+        assertUpdate("DELETE FROM " + tableName + " WHERE id = 1", 1);
+        assertQuery("SELECT * FROM " + tableName, "VALUES (2, 'world')");
+
+        String sqlStatement = (String) computeScalar("SHOW CREATE TABLE " + tableName);
+        String tableProperties = sqlStatement.split("\\)\\s*WITH\\s*\\(")[1];
+        // empty partition keys
+        assertTableProperty(tableProperties, "partition_by_hash_columns", Pattern.quote("ARRAY[]"));
+        assertTableProperty(tableProperties, "partition_by_range_columns", Pattern.quote("ARRAY[]"));
```

Comment:
Can we check the output of `SHOW CREATE ..` and assert that String ?

Reply:
As a follow-up we can also introduce a SystemTable for Kudu and simplify these further instead of parsing the String.